integrations/block-data-api.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads VIP Block Data REST API.
+ *
+ * @private
+ */
+class BlockDataApi extends Integration {
+
+	/**
+	 * The version of the Block Data API plugin to load, that's set to the latest version.
+	 * This should be higher than the lowestVersion set in https://github.com/Automattic/vip-go-mu-plugins-ext/blob/trunk/config.json#L63
+	 * 
+	 * @var string
+	 */
+	protected string $version = '1.0';
```

Comment:
This now mismatches the path in ext (it.s currently there as 1.0.0 (and 1.0.1 in the unmerged PR).

Reply:
Tested locally with the latest changes in https://github.com/Automattic/vip-go-mu-plugins-ext/pull/35, -ext should pull `vip-block-data-api-1.0` for any `1.0.x` tag. -ext will be merged first.